docs/userguide.md

```diff
@@ -77,6 +78,25 @@ kubectl get all --all-namespaces
 Once all pods are running you can move on to deploy the cluster registry and
 federation-v2 control plane.
 
+#### Google Kubernetes Engine (GKE)
```

Comment:
I suggest moving all of this content to the new `gke.md` doc. Then we can just add a new `Cloud Provider` setup section that lists any of the cloud providers for which we have instructions, with links to their respective docs. Otherwise this user guide could get quite exhaustive if we start providing setup instructions for each provider.